<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Campaign Creator (c) PMC</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Campaign Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CampaignName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>XXX type campaign name here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>550</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Campaign</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="StatusBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>363</y>
     <width>621</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="weaponPool">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>weaponPool</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="OverviewText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>761</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>type overview text here</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Campaign Overview</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numberOfMissions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of missions</string>
   </property>
  </widget>
  <widget class="QComboBox" name="terrainName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Utes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chernarus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Takistan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Desert_E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shapur_BAF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>provinggrounds_pmc</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMC_51km_Desert</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMC_Rattler</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMC_Rugen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>isladuala</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="missionTagName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pmc_cmp_mission</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mission Class and File Name TAG</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="openingCutscene">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Opening Cutscene</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cutsceneEveryMission">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Cutscene between every mission</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="endingCutscene">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ending Cutscene</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
